src/App.js

```diff
@@ -1,23 +1,14 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
+// import logo from "./logo.svg";
 import "./App.css";
+import WebcamFeed from "./components/webcamFeed";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>asanaApp lives here</p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        Here's the webcam
```

Comment:
Do we need a <WebcamFeed> here?

Reply:
React components only have one <> component tag. Only need the one <WebcamFeed /> here.

Reply:
nope